app/ai-health-coach/2025/09/index.md

```diff
@@ -0,0 +1,119 @@
+---
+title: AI Health Coach discovery summary
+description: An overview of the research, analysis, and conclusions from discovery phase
+date: 2025-09-24
+tags:
+  - discovery
+  - AI
+---
+
+This discovery explored the challenges faced by users of prevention and behaviour change services. It also examined whether these challenges could be addressed through the application of Artificial Intelligence, with a particular focus on AI’s potential to provide health coaching.
+
+## Our problem statement
+
+> Users of prevention services often experience support that feels disconnected or incomplete, and struggle with tools that only help with one aspect of their health – rather than supporting their broader, interconnected needs. While AI has the potential to improve personalisation and connection across services, it’s currently unclear how best to use AI to support across all aspects of someone’s prevention journey.
+
+## Our users
+
+Our users are those participating in (or could benefit from) healthy behaviour change or prevention services. Our review of existing research focused on users at various stages of that journey:
+
+- Better Health users and NHS App users
+- adults in the UK aged between 18 and 60, who were mostly from deprived areas (socioeconomic groups C2, D, E)
+- people with at least one raised health risk factor (being overweight, being inactive, smoking, heavier drinking)
+
+Other teams across NHS England and Department for Health and Social Care had already carried out extensive research to understand the pain points and barriers across this journey, so we drew upon these rather than conducting new primary research.
+
+## Research and analysis
+
+We identified pain points and user needs, as well as assess how these are being met by the existing landscape. We explored:
```

Comment:
This sentence feels slightly ungrammatical with a mix of tenses ('identified' and 'asses')?

Could it be something like this:

```suggestion
We identified pain points and user needs, and assessed how these are being met by the existing landscape. We explored:
```